src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/extension/BuildCardExtension/build-card-template-onclicks.js

```diff
@@ -0,0 +1,21 @@
+Behaviour.specify(".progress-console-onclick", 'progress-bar-click', 0, function (progressBarElement) {
```

Comment:
I'd suggest to make the class more more specific, perhaps include plugin name. E.g. `bpp-progress-console-onclick`. Same for the ID argument, what I've seen in the ecosystem a lot is people prepending a name of a Java class that a view belongs to. E.g. here I'd go for something like `BuildCardExtension_progressBarClick`.